packages/pigeon/README.md

```diff
@@ -5,12 +5,11 @@ host platform type-safe, easier and faster.
 
 ## Supported Platforms
 
-Currently Pigeon supports generating Objective-C and Swift code
-for usage on iOS, Java and Kotlin code for Android, 
-and has experimental support for C++ for Windows.
-The Objective-C code is
-[accessible to Swift](https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/importing_objective-c_into_swift)
-and the Java code is accessible to Kotlin.
```

Comment:
I removed this sentence since it was there to reassure people that Pigeon could work with Swift/Kotlin projects back when it didn't support those languages; it's no longer useful to mention.